src/output/plugins/httpd/HttpdClient.cxx

```diff
@@ -102,6 +112,15 @@ HttpdClient::HandleLine(const char *line) noexcept
 			return true;
 		}
 
+		if (StringEqualsCaseASCII(line, "Authorization: Basic ", 21)) {
```

Comment:
Where does the `21` come from? This coding style is fragile, because who really verifies that this string literal has 21 characters?
Try this:
```
if (const char *b64 = StringAfterPrefixIgnoreCase(line, "Authorization: Basic ")) {
```
This lets the compiler figure out the size of the string literal, no need to do it manually, nothing fragile here.

Reply:
Done.